webapi/src/test/scala/org/knora/webapi/e2e/ClientTestDataCollector.scala

```diff
@@ -8,53 +8,51 @@ package org.knora.webapi.e2e
 import org.knora.webapi.exceptions.TestConfigurationException
 import org.knora.webapi.settings.KnoraSettingsImpl
 import redis.clients.jedis.{Jedis, JedisPool, JedisPoolConfig}
```

Comment:
I see that you deleted redis related things, do we still need the redis library here?